holopy/inference/third_party/nmpfit.py

```diff
@@ -1399,15 +1399,15 @@ def iterfunct(myfunct, p, iter, fnorm, functkw=None,
 
                 ## Fill in actual covariance matrix, accounting for fixed
                 ## parameters.
-                self.covar = numpy.zeros([nn, nn], numpy.float)
+                self.covar = numpy.zeros([nn, nn], float)
```

Comment:
Just out of curiosity, why did you delete the explicit `numpy` module name? Was it causing a bug?

Reply:
deprecated as of numpy 1.20. Apparently it just points to build in python float anyway.

Reply:
`np.float64` is still a thing, but `np.float` was just another name for `float` and they're killing it. Same with `int`.

Reply:
(I'm just curious, no need to change anything.)